webview/src/views/main.svelte

```diff
@@ -41,16 +50,29 @@
 
 <section>
   <p>Debug by ID</p>
-  <form on:submit|once|preventDefault={startFromWorkflowId}>
-    <input type="text" placeholder="namespace (default)" name="namespace" />
-    <input type="text" required placeholder="Workflow ID *" name="workflowId" />
-    <input type="text" placeholder="Run ID" name="runId" />
-    <input type="submit" value="Start" />
+  <form id="debug-by-id-form">
+    <vscode-text-field type="text" placeholder="Namespace (default)" name="namespace" />
+    <vscode-text-field type="text" required placeholder="Workflow ID *" name="workflowId" />
+    <vscode-text-field type="text" placeholder="Run ID" name="runId" />
+    <vscode-button on:click={startFromWorkflowId}>Start</vscode-button>
```

Comment:
**Note:** I'd like to revisit using a `submit` button here in order to take advantage of browser validation for `required` inputs. 

Adding `<vscode-button type-"submit"/>` is not currently working and doesn't trigger a form `on:submit` event when clicked  😞

Reply:
Yes please for using submit buttons for forms, people want to use their keyboards to submit.

Reply:
I'd consider this critical.

Reply:
Since `<vscode-button type-"submit"/>` is not triggering an `on:submit` event*, I've gone ahead and created a custom `SubmitButton` component for now that is styled the same as a `vscode-button` but with which the default form behavior works. 

I've added a `TODO` in the component to hopefully one day use `vscode-button` again, but until there is better support for form behavior in the ui toolkit this seems like the best solution to utilize form behavior (see https://github.com/microsoft/vscode-webview-ui-toolkit/issues/371 and https://github.com/microsoft/vscode-webview-ui-toolkit/issues/395).

*the `on:submit` isn't triggered even with `type="submit"` unless an `on:click` is added to the button in which case the function for submit is triggered twice.

Reply:
Since `<vscode-button type-"submit"/>` is not triggering an `on:submit` event*, I've gone ahead and created a custom `SubmitButton` component for now that is styled the same as a `vscode-button` but with which the default form behavior works. 

I've added a `TODO` in the component to hopefully one day use `vscode-button` again, but until there is better support for form behavior in the ui toolkit (or I can figure out why it isn't working as expected) this seems like the best solution to utilize form behavior.

*the `on:submit` isn't triggered even with `type="submit"` unless an `on:click` is added to the button in which case the function for submit is triggered twice.

Reply:
Thanks!